### DISCUSSION TITLE
What's the correct way to add context tags/badges?

### DISCUSSION THREAD

**Discussion by araujoarthur** [squidfunk/mkdocs-material#7137] [Q&A] [answered]
Hello! I've been searching for a way to add this to my docs:

![image](https://github.com/squidfunk/mkdocs-material/assets/28057772/2d96666a-4afe-41f2-90b9-8d930ad70e1a)

But the closest approach I got up to now was to add icons and text side by side:

![image](https://github.com/squidfunk/mkdocs-material/assets/28057772/a80a6468-9d6c-44fa-8554-f73967045860)

**Comment by alexvoss** [comment, marked as answer]
You can have a look at how this is done in the Material for MkDocs documentation in [src/overrides/hooks/shortcodes.py](https://github.com/squidfunk/mkdocs-material/blob/5cb3117f50c78abfef6817b72cb09910decd272b/src/overrides/hooks/shortcodes.py). The corresponding CSS is in [src/overrides/assets/stylesheets/custom/_typeset.scss](https://github.com/squidfunk/mkdocs-material/blob/5cb3117f50c78abfef6817b72cb09910decd272b/src/overrides/assets/stylesheets/custom/_typeset.scss#L97).

Note that this is not part of Material for MkDocs but a customization for our own documentation. To find previous answers to the same question, search for "convention" or "shortcodes" in the discussions.

**Reply by araujoarthur** [reply]
Thank you :)
